syntax = "proto3";

package recommend_api.v1;

service RecommenderService {
    rpc Recommend (RecommendRequest) returns (RecommendResponse);
}

message RecommendRequest {
    repeated int32 item_ids = 1;
}

message RecommendResponse {
    repeated int32 item_ids = 1;
}